src/rgw/rgw_sal_motr.cc

```diff
@@ -2054,6 +2071,7 @@ void MotrAtomicWriter::cleanup()
   m0_bufvec_free2(&buf);
   acc_data.clear();
   obj.close_mobj();
+  old_obj.close_mobj();
```

Comment:
Is there any need to close the old_obj ? I suppose it was not opened at all.

Reply:
Right, mobj is not opened before we are actually trying to delete it. I have added for the sake of cleanup.